definitions/subworkflows/pvacseq.wdl

```diff
@@ -204,19 +199,35 @@ workflow pvacseq {
     genotype_fields=variants_to_table_genotype_fields
   }
 
-  call avftt.addVepFieldsToTable {
-    input:
-    vcf=index.indexed_vcf,
-    vep_fields=vep_to_table_fields,
-    tsv=variantsToTable.variants_tsv,
-    prefix=prefix,
-    preferred_transcripts_tsv=pvacseqAggregatedReportToPreferredTranscriptsList.preferred_transcripts_tsv
+  if (length(select_all([ps.mhc_i_aggregated_report, ps.mhc_ii_aggregated_report])) > 0 ) {
+      call ptl.pvacseqAggregatedReportToPreferredTranscriptsList as pvacseqAggregatedReportToPreferredTranscriptsList {
+        input:
+        pvacseq_aggregated_report=select_first([ps.mhc_i_aggregated_report, ps.mhc_ii_aggregated_report])
+      }
+
+      call avftt.addVepFieldsToTable as addVepFieldsToTableWithPreferredTranscriptsTsv{
+        input:
+        vcf=index.indexed_vcf,
+        vep_fields=vep_to_table_fields,
+        tsv=variantsToTable.variants_tsv,
+        prefix=prefix,
+        preferred_transcripts_tsv=pvacseqAggregatedReportToPreferredTranscriptsList.preferred_transcripts_tsv
+      }
+  }
+  if (length(select_all([ps.mhc_i_aggregated_report, ps.mhc_ii_aggregated_report])) == 0 ) {
+      call avftt.addVepFieldsToTable as addVepFieldsToTableWithoutPreferredTranscriptsTsv{
```

Comment:
Is this call the same as the one above except for the `preferred_transcripts_tsv` parameter only being present above?  If so you could possibly pull this step out from the conditional and let it be an optional input based on whether or not the previous step ran.  (Basically the same as the prior `addVepFieldsToTable` these two steps are replacing).